agents/src/llm/function_context.ts

```diff
@@ -133,7 +133,7 @@ export const oaiBuildFunctionInfo = (
 
   return {
     name: fncName,
-    func: fncCtx[fncName],
+    func: fncCtx[fncName]!,
```

Comment:
a temp var here could also make sense to use it both for the checkin in L130 and here